controllers/endpoint/endpoints_controller_test.go

```diff
@@ -177,7 +177,7 @@ func testWeederDedicatedEnvTest(t *testing.T) {
 func testOnlyCLBFPodDeletion(ctx context.Context, _ context.CancelFunc, g *WithT, reconciler *Reconciler, namespace string) {
 	createEp(ctx, g, reconciler, namespace, true)
 	pC := newPod(crashingPod, namespace, "node-0", correctLabels)
-	pH := newPod(healthyPod, namespace, "node-0", correctLabels)
+	pH := newPod(healthyPod, namespace, "node-1", correctLabels)
```

Comment:
It's not clear when I read this test as it use acronym `CLBF` without explaining what it means. 
Also we move to a new node for healthy pod, but then we should update the description above of the test to highlight this change.

Reply:
This was an existing method name that was not changed But you are right that we can have a better name. CLBF means crash loopbackoff